packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -87,19 +87,12 @@ cbp-structured-list {
     font-style: italic;
   }
 
-
- 
-  [slot=cbp-structured-list-footer] {
-    color: var(--cbp-structured-list-footer-color);
-    background-color: var(--cbp-structured-list-footer-color-background);
-  }
-
   &[striped] [role=list] > *:nth-child(even) {
     background-color: var(--cbp-structured-list-color-striped-background);
   }
 
   // Style all list items regardless of the element used to wrap/define them (don't leave this to cbp-structured-list-item)
-  div[role=list] > * {
+  div[role=list] > *:not(cbp-action-bar) {
```

Comment:
this addition shouldn't be needed because the action bar is not inside of the list anyway.

Reply:
without this the display block was breaking layout of the action bar on the structured list footer. can look for separate solution but this was least verbose way i could think of ATM

Reply:
updating to use the slot=cbp-structured-list-footer resolved this better, removing this :not()